src/items/item-persistence.js

```diff
@@ -617,13 +644,64 @@ class ItemPersistence {
    *
    * @param {(time.ZonedDateTime | Date)} begin the point in time from which to start the average
    * @param {(time.ZonedDateTime | Date)} end the point in time to which to start the average
+   * @param {RiemannType} [riemannType] optional Riemann approximation type to calculate the integral approximation
+   *                                    (<code>LEFT</code>, <code>RIGHT</code>, <code>TRAPEZOIDAL</code>, <code>MIDPOINT</code>),
+   *                                    default <code>LEFT</code>
    * @param {string} [serviceId] optional persistence service ID, if omitted, the default persistence service will be used
    * @returns {(PersistedState | null)} the average value between <code>begin</code> and <code>end</code> as {@link items.PersistedState} or <code>null</code> if no states could be found
    */
-  averageBetween (begin, end, serviceId) {
+  averageBetween (begin, end, riemannType, serviceId) {
     return _persistedStateOrNull(PersistenceExtensions.averageBetween(this.rawItem, ...arguments));
   }
 
+  /**
+   * Gets the RiemannSum of the states of a given Item since a certain point in time, time is calculated in seconds.
+   *
+   * @example
```

Comment:
I would rather have an example with power measurement and energy consumption calculation, 
this example IMO is not really "meaningful".